iFixit/react-commerce
# Vulcan: Add breadcrumbs to page

This adds the breadcrumbs to the page. It's using the existing API we added in https://github.com/iFixit/ifixit/pull/46347.

## QA Notes
Please check that the breadcrumbs seem reasonable and look as-expected.

Please also check that the page looks reasonably like the mockup at a mobile size.

Closes https://github.com/iFixit/ifixit/issues/46770
Connects https://github.com/iFixit/ifixit/issues/46879
